content/docs/07-git-cheats.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Git cheats
+---
+
+### Github commands and definitions 
+
+Basics
```

Comment:
Append `### ` If you want these headings to be linked in the side bar. Up to you.

Reply:
Sounds good I'll add that!